app/views/plantings/index.html.haml

```diff
@@ -1,21 +1,10 @@
 - content_for :title, @owner ? t('.title.owner_plantings', owner: @owner) : @crop ? t('.title.crop_plantings', crop: @crop.name) : t('.title.default')
+
+= render 'nav', owner: @owner, show_all: @show_all
```

Comment:
This makes the navigation look consistent across the 3 ways to view a plantings#index.
I want to do the same to gardens and crops too